support-docs/examples.md

```diff
@@ -1281,33 +1204,41 @@ USB-C PD Controller Example
         "unitFactor": 1e-3,
         "valueDefined": true
     },
-    "shutDownCurrent": {
-        "siUnit": "amp",
-        "absoluteValue": 10,
-        "unitText": "uA",
-        "unitFactor": 1e-6,
-        "valueDefined": true
-    },
-    "idleCurrent": {
-        "siUnit": "amp",
-        "absoluteValue": 5,
-        "unitText": "mA",
-        "unitFactor": 1e-3,
-        "valueDefined": true
-    },
-    "vbusOvervoltageProtectionThresholdRising": {
-        "siUnit": "volt",
-        "absoluteValue": 6.3,
-        "unitText": "v",
-        "unitFactor": 1,
-        "valueDefined": true
+    "currentConsumption": {
+        "supplyName": "pwr1",
+        "shutDownCurrent": {
+            "typValue": 10,
+            "siUnit": "amp",
+            "unitText": "uA",
+            "unitFactor": 1e-6,
+            "valueDefined": true
+        },
+        "idleCurrent": {
+            "typValue": 5,
+            "siUnit": "amp",
+            "unitText": "mA",
+            "unitFactor": 1e-3,
+            "valueDefined": true
+         }
     },
-    "vbusOvervoltageProtectionThresholdFalling": {
-        "siUnit": "volt",
-        "absoluteValue": 6.1,
-        "unitText": "v",
-        "unitFactor": 1,
-        "valueDefined": true
+    "componentProtectionThresholds": {
```

Comment:
as mentioned in the spec, I think this should be:

vbusProtectionThresholds

Reply:
See above on changes within "powerSupplyProtection" object to make it an array.